README.md

```diff
@@ -325,6 +325,29 @@ type MenuItem = {
 DashKit.setSettings({menu: [] as Array<MenuItem>});
 ```
 
+| Name                                           | Description           |
```

Comment:
Can you add some caption? Like CSS Api. And would be good to add some recommendations how to use it.